src/constants/file-location-path.constant.ts

```diff
@@ -1,3 +1,4 @@
 export const FILE_LOCATION_PATH = {
-  REGEX: /^[\w\-:/\\()\s]+$/,
+  // [a-fA-F\d]{24} matches a MongoDB hex, which will be the name of the first folder in the path.
+  REGEX: /^[a-fA-F\d]{24}(\/[\w\-:\\()\s]+)*$/,
```

Comment:
It looks like we've accidentally changed the `REGEX` here

Reply:
The linter uses the [safe-regex](https://www.npmjs.com/package/safe-regex) package which can give false positives as it's a simple (but quick) check. I've put the regex into another checker (https://devina.io/redos-checker, which uses [recheck](https://github.com/makenowjust-labs/recheck)) which says that it's safe. I'll bring this up with the team to see if we want to move to a better but slower regex checker